data/json/items/chemicals_and_resources.json

```diff
@@ -890,5 +890,24 @@
     "weight": 212,
     "volume": 1,
     "to_hit": -2
-    }
+    },{
+    "type" : "COMESTIBLE",
```

Comment:
This possibly stems back to the nethack tradition of risky item discovery. That mechanic doesn't apply here - accidentally drinking sewage or setting of a stick of dynamite instead of turning on a flashlight is a UI problem not a feature.